docs/releases/1.9.1.txt

```diff
@@ -82,4 +82,4 @@ Bugfixes
 
 * Fixed a crash in ``QuerySet.values()/values_list()`` after an ``annotate()``
   and ``order_by()`` when ``values()/values_list()`` includes a field not in
-  the ``order_by()`` (:ticket:`25316`).
+  the ``order_by()`` (:ticket:`25316`).
```

Comment:
Revert this file -- I guess your editor might have removed a trailing newline or something.